live-examples/js-examples/promise/promise-constructor.html

```diff
@@ -1,9 +1,23 @@
 <pre>
 <code id="static-js">var promise1 = new Promise(function(resolve, reject) {
-  setTimeout(resolve, 100, 'foo');
+  setTimeout(function() { 
+    resolve('foo');
+  }, 100);
 });
 
+
+// How to show the value of promise1: 
+
+promise1.then(value => console.log(value));
+//=> "foo"
+
+// A Promise's value can ONLY be accessed via a `.then()` callback function:
+// promise1.then(function(value) {/* handle value in function here */})
+
+// The following example **won't work**:
+// Reading Synchronously:
```

Comment:
I don't think these comments are needed. This is an example, it is not the documentation. We should be very sparing with comments in these examples IMO - we should aim to show not tell. There are plenty of words in the rest of the page for people to read.